mock-recipe-server/api_index.html

```diff
@@ -148,11 +148,9 @@
         let url = new URL(`/${testcase.dataset.name}/api/v1`, window.location);
         testcase.querySelector('.url').textContent = url.href;
         testcase.querySelector('.browser-code').value = (
-          `Cu.import('resource://shield-recipe-client/lib/Storage.jsm');\n` +
-          `Storage.clearAllStorage();\n` +
-          `Services.prefs.setCharPref("extensions.shield-recipe-client.api_url", "${url.href}");\n` +
+          `Cu.unload('resource://shield-recipe-client/lib/RecipeRunner.jsm');\n` +
```

Comment:
Instead of unloading aggressively, could we `delete RecipeRunner;`, so the reference held by the console is dropped immediately? Then the normal add-on reload behavior should work correctly.

Alternatively, we could wrap this in an iife, so that all our references fall out of scope immediately.